policies/feature-flags/feature-flags-age.py

```
@@ -0,0 +1,61 @@
+def main(node=None):
+    """Check that feature flags are not older than the configured threshold."""
+    c = Check("feature-flags-age", "Feature flags should not exceed age threshold", node=node)
+    with c:
+        # Get the maximum age threshold from configuration
+        max_days_str = variable_or_default("max_days", "90")
+        try:
+            max_days = int(max_days_str)
+        except ValueError:
+            raise ValueError(
+                f"Policy misconfiguration: 'max_days' must be an integer, got '{max_days_str}'"
+            )
```

Ruff (0.14.14)

[warning] Within an `except` clause, raise exceptions with `raise ... from err` or `raise ... from None` to distinguish them from errors in exception handling

(B904)